library/elliptics.h

```diff
@@ -1035,8 +1035,7 @@ static inline int dnet_read_ll(int fd, char *data, size_t size, off_t offset)
 /*
  * Watermarks for number of bytes written into the wire
  */
-#define DNET_SERVER_SEND_WATERMARK_HIGH		(30*1024*1024*1024L)
-#define DNET_SERVER_SEND_WATERMARK_LOW		DNET_SERVER_SEND_WATERMARK_HIGH / 2
+#define DNET_SERVER_SEND_WATERMARK_HIGH		(100*1024*1024L)
```

Comment:
It is unlikely that you want to fully utilize any network in the first place, since you also need a 'space' in the wire for client requests and optionally other backend recovery.

But nevertheless 100 megabytes of pending unacknowledged data is quite large chunk and it is calculated after key has been submitted into the wire, i.e. iterator sends current key and then sleeps if number of unacknowledged bytes is higher than this limit. If you have fast network and remote storage and big keys, this limit can be exceeded.